src/sentry/static/sentry/app/components/events/interfaces/frame/frame.tsx

```diff
@@ -18,14 +18,37 @@ import {SymbolicatorStatus} from 'app/components/events/interfaces/types';
 import {combineStatus} from 'app/components/events/interfaces/debugMeta/utils';
 import {IconRefresh, IconAdd, IconSubtract, IconQuestion, IconWarning} from 'app/icons';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {Frame as FrameType, SentryAppComponent, PlatformType} from 'app/types';
+import DebugImage from 'app/components/events/interfaces/debugMeta/debugImage';
 
 import FrameDefaultTitle from './frameDefaultTitle';
 import FrameContext from './frameContext';
 import FrameFunctionName from './frameFunctionName';
 import {getPlatform} from './utils';
 
-export class Frame extends React.Component {
-  static propTypes = {
+type Props = {
+  data: FrameType;
+  nextFrame: FrameType;
+  prevFrame: FrameType;
+  platform: PlatformType;
+  isExpanded: boolean;
+  emptySourceNotation: boolean;
+  isOnlyFrame: boolean;
+  timesRepeated: number;
+  registers: {[key: string]: string};
+  components: Array<SentryAppComponent>;
+  showingAbsoluteAddress: boolean;
+  onAddressToggle: () => void;
+  image: React.ComponentProps<typeof DebugImage>['image'];
+  maxLengthOfRelativeAddress: number;
+};
+
+type State = {
+  isExpanded: boolean;
+};
+
+export class Frame extends React.Component<Props, State> {
+  static propTypes: any = {
```

Comment:
Do you still need this propTypes?

Reply:
@priscilawebdev Yea, this component is still used in a few Javascript files.